ROADMAP.md

```diff
@@ -0,0 +1,24 @@
+# Roadmap
+The roadmap is for information only. The listed items may be added, removed, or changed due to priority change, dependency readiness and customer demands, etc.
+### 2022
+
+Teams Toolkit for Visual Studio Code and Teams Toolkit CLI are GA in May 2022, Teams Toolkit for Visual Studio will be GA in August 2022.
```

Comment:
GA date for Teams Toolkit for Visual Studio Code and Teams Toolkit CLI - May 2022
GA date for Teams Toolkit for Visual Studio - August 2022

Reply:
content will be replaced by John's final version tomorrow, so can ignore things in ROADMAP.md